src/play-area/play-area-handler.js

```diff
@@ -12,25 +12,33 @@ var UpdateBufferData = function UpdateBufferData(index, value) {
 module.exports = function PlayAreaHandler() {
 
     var updateBuffer = [];
+    var shapeToGridConverter = ShapeToGridConverter.createShapeToGridConverter();
 
     function applyShape(playArea, shape, value) {
-        var points = ShapeToGridConverter().convert(shape, playArea);
+        var points = shapeToGridConverter.convert(shape, playArea);
         var grid = playArea.grid;
+        var changedData = [];
         for (var i = 0; i < points.length; i++) {
             // Buffer should only be updated when a value has changed
             if (grid[points[i]] !== value) {
                 grid[points[i]] = value;
-                updateBuffer.push(UpdateBufferData(points[i], value));
+                changedData.push(UpdateBufferData(points[i], value));
             }
         }
+        updateBuffer = updateBuffer.concat(changedData);
+        return changedData;
+    }
+
+    function getCellValue(playArea, index) {
```

Comment:
Den här känns aningen meningslös, eftersom `playArea.grid[index]` är ett annat "namn" för denna metod. Hade varit en mer välmotiverad metod ifall den abstraherar användaren så att man skickar in en point/koordinat eller liknande.